ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -29,6 +29,7 @@ dependencies:
   - gcsfs=0.6.0
   - jinja2=2.11
   - lxml=4.5.0
+  - markupsafe=2.0.1
```

Comment:
Could you add a note that this is a Jinja dependency not necessarily a pandas one and why it was pinned?

Reply:
Done, we can not really add comments inline, because the version sync script can not deal with them. Added it in the optional dependencies header line

Reply:
Great, thanks!